lib/esp32.toit

```diff
@@ -100,3 +100,19 @@ image_config_ -> ByteArray:
 
 set_real_time_clock_ seconds/int ns/int -> none:
   #primitive.core.set_real_time_clock
+
+/**
+Sets the RTC user data to the given $data starting from the given $offset.
```

Comment:
Add some explanation what RTC is.

Reply:
Added advanced section with a bit of explanation.